.github/workflows/r_package.yml

```diff
@@ -308,7 +308,7 @@ jobs:
           #
           wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
           #
-          add-apt-repository "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-15 main"
+          add-apt-repository "deb http://apt.llvm.org/unstable/ llvm-toolchain main"
```

Comment:
This is from #5689, just to unblock CI on this PR. It'll drop out of the diff once that's merged.